cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -87,6 +91,14 @@ bool CobTwistController::initialize()
         ROS_WARN_STREAM("Parameter 'max_vel_rot_base' not set. Using default: " << twistControllerParams_.max_vel_rot_base);
     }
 
+    // Frames of Interest
+    if(!nh_twist.getParam("collision_check_frames", twistControllerParams_.collision_check_frames))
+    {
+        ROS_WARN_STREAM("Parameter vector 'collision_check_frames' not set. Using default: 3rd link and 5th link of manipulator.");
+        twistControllerParams_.collision_check_frames.push_back(robo_namespace + "_3_link");
+        twistControllerParams_.collision_check_frames.push_back(robo_namespace + "_5_link");
```

Comment:
I think we should rather disable the constraint, then providing default values...

because this node can also be used with e.g. the universal robot where the joint and links are not numbered but have totally different names...and I guess then you will get tf_transformation problems lateron...